joint-discovery-gatk4-fc.wdl

```diff
@@ -0,0 +1,896 @@
+    File tranches = "${tranches_filename}"
+  }
+}
+
+task SNPsVariantRecalibratorCreateModel {
+  String recalibration_filename
+  String tranches_filename
+  Int downsampleFactor
+  String model_report_filename
+
+  Array[String] recalibration_tranche_values
+  Array[String] recalibration_annotation_values
+
+  File sites_only_variant_filtered_vcf
+  File sites_only_variant_filtered_vcf_index
+
+  File hapmap_resource_vcf
+  File omni_resource_vcf
+  File one_thousand_genomes_resource_vcf
+  File dbsnp_resource_vcf
+  File hapmap_resource_vcf_index
+  File omni_resource_vcf_index
+  File one_thousand_genomes_resource_vcf_index
+  File dbsnp_resource_vcf_index
+
+  String docker_image
+  Int disk_size
+
+  command {
+    /gatk/gatk-launch --javaOptions "-Xmx100g -Xms100g" \
+      VariantRecalibrator \
+      -V ${sites_only_variant_filtered_vcf} \
+      -O ${recalibration_filename} \
+      --tranchesFile ${tranches_filename} \
+      -allPoly \
+      -tranche ${sep=' -tranche ' recalibration_tranche_values} \
+      -an ${sep=' -an ' recalibration_annotation_values} \
+      -mode SNP \
+      -sampleEvery ${downsampleFactor} \
+      --output_model ${model_report_filename} \
+      --maxGaussians 6 \
+      -resource hapmap,known=false,training=true,truth=true,prior=15:${hapmap_resource_vcf} \
+      -resource omni,known=false,training=true,truth=true,prior=12:${omni_resource_vcf} \
+      -resource 1000G,known=false,training=true,truth=false,prior=10:${one_thousand_genomes_resource_vcf} \
+      -resource dbsnp,known=true,training=false,truth=false,prior=7:${dbsnp_resource_vcf}
+  }
+  runtime {
+    docker: docker_image
+    memory: "104 GB"
+    cpu: "2"
+    disks: "local-disk " + disk_size + " HDD"
+    preemptible: 5
+  }
+  output {
+    File model_report = "${model_report_filename}"
+  }
+}
+
+task SNPsVariantRecalibratorScattered {
+  String recalibration_filename
+  String tranches_filename
+  File model_report
+
+  Array[String] recalibration_tranche_values
+  Array[String] recalibration_annotation_values
+
+  File sites_only_variant_filtered_vcf
+  File sites_only_variant_filtered_vcf_index
+
+  File hapmap_resource_vcf
+  File omni_resource_vcf
+  File one_thousand_genomes_resource_vcf
+  File dbsnp_resource_vcf
+  File hapmap_resource_vcf_index
+  File omni_resource_vcf_index
+  File one_thousand_genomes_resource_vcf_index
+  File dbsnp_resource_vcf_index
+
+  String docker_image
+  Int disk_size
+
+  command {
+    /gatk/gatk-launch --javaOptions "-Xmx3g -Xms3g" \
+      VariantRecalibrator \
+      -V ${sites_only_variant_filtered_vcf} \
+      -O ${recalibration_filename} \
+      --tranchesFile ${tranches_filename} \
+      -allPoly \
+      -tranche ${sep=' -tranche ' recalibration_tranche_values} \
+      -an ${sep=' -an ' recalibration_annotation_values} \
+      -mode SNP \
+      --input_model ${model_report} \
+      -scatterTranches \
+      --maxGaussians 1 \
```

Comment:
this has changed from 6 -> 1 - https://github.com/broadinstitute/dsde-pipelines/blob/develop/genomes_in_the_cloud/joint_genotyping_workflow/JointGenotypingWf.wdl#L530

Reply:
Left over edits from the using small input files. will fix this here and double check the method in FC.